### PR TITLE
Update open question sections

### PR DESCRIPTION
**Pull request by dot-miniscule**
Adds a section on updating, after much kerfuffling.

### CODE REVIEW

File: phase-4-explainer.md [hunk 1]
```diff
@@ -153,42 +150,80 @@ interface ComputedAccessibleNode {
...
-    readonly attribute float? valueMin;
-    readonly attribute float? valueMax;
-
     [CallWith=ScriptState] Promise ensureUpToDate();
 };
 ```
 
-TODO(meredithl): insert link to full list of potential properties in spec.
-
+See the [proposed Phase 4 Spec](spec/phase4.html) for a full list of properties available to be implemented.
```
Comment (dot-miniscule) [inline phase-4-explainer.md:176]: This should be a link to probably the hosted spec I imagine?

File: phase-4-explainer.md [hunk 2]
```diff
@@ -153,42 +150,80 @@ interface ComputedAccessibleNode {
...
 
 ### 2. Rejection Case
-If we are returning a promise when requesting a `ComputedAccessibleNode`, what is the rejection case?
+If we are returning a promise when requesting a `ComputedAccessibleNode`, what is the rejection case? What does it mean for a `ComputedAccessibleNode` to be rejected? This may be discovered as other open questions are answered.
 
 ### 3. Traversing across Accessible Trees
 Currently Phase 4 is proposed as containing `ComputedAccessibleNode`s within a single Frame, but in future the capability to explore from one Accessible Tree in one Frame to another.
-- TODO: Use cases for this?
-- Considerations? How to get from one "root" to "another"?
+- Considerations?
```
Comment (dot-miniscule) [inline phase-4-explainer.md:183]: This section might need some more expansion, I don't really understand this section tbh.

File: phase-4-explainer.md [hunk 3]
```diff
@@ -153,42 +150,80 @@ interface ComputedAccessibleNode {
...
   - What is the relation between Frame roots? Siblings?
-  - What owns each Frame? roots of tree's in different frame?
-  - Is each tree still owned by the frame?
 
 ### 4. Non-dom Nodes
-We may want the ability to traverse to non-DOM nodes through the `ComputedAccessibleNode` API. For example, traversing to the root of a page, or to [virtual accessibility nodes](explainer.md#phase-3-virtual-accessibility-nodes).
+We may want the ability to traverse to non-DOM nodes through the `ComputedAccessibleNode` API. How should `ComputedAccessibleNode`s fit into the notion of virtual nodes and Shadow DOM? A `ComputedAccessibleNode` can only be requested from the `Window` via an `Element`, but we may want to expose a way to traverse to `ComputedAccessibleNode`s for non-DOM nodes, such as pseudo-elements, or the root of the page. Tree walking should also be inclusive of [virtual accessibility nodes](explainer.md#phase-3-virtual-accessibility-nodes). Should `ComputedAccessibleNode`s allow us to traverse into Shadow-DOM? If so, should we be able to traverse into a closed Shadow-DOM? Is this a good candidate for the [rejection case](##2-rejection-case)?
```
Comment (dot-miniscule) [inline phase-4-explainer.md:188]: May be wildly misunderstanding shadow DOM here.

File: phase-4-explainer.md (hunk 1 above)
Comment (alice) [reply phase-4-explainer.md:176]: This is fine for now.

File: phase-4-explainer.md (hunk 2 above)
Comment (alice) [reply phase-4-explainer.md:183]: We can chat about it later, along with all that promise resolver fun.

File: phase-4-explainer.md (hunk 3 above)
Comment (alice) [reply phase-4-explainer.md:188]: ¯\\_(ツ)\_/¯

File: phase-4-explainer.md [hunk 4]
```diff
@@ -153,42 +150,80 @@ interface ComputedAccessibleNode {
...
+    childList: true,
+    subtree: true,
+    attributeOldValue: true,
+    characterDataOldValue: true
+});
+
+mutationObserver.disconnect();
+```
+
+What other options exist for updating `ComputedAccessibleNode`s?
```
Comment (alice) [inline phase-4-explainer.md]: I'd #### this

**Review by alice — APPROVED** [review]

File: phase-4-explainer.md (hunk 3 above)
Comment (dot-miniscule) [reply phase-4-explainer.md:188]: :woman_shrugging:

**Pull request merged**